tests/main/help/task.yaml

```diff
@@ -1,5 +1,5 @@
 summary: Check commands help
-systems: [fedora-*, opensuse-*]
+systems: [fedora-*, opensuse-*, ubuntu-*, debian-*]
```

Comment:
Silly question - I think thats all the systems we have, no? So could we just delete the `systems:` line entirely?